scripts/rebuilder.py

```diff
@@ -14,6 +15,12 @@
 from munch import Munch
 
 
+def get_rawhide_tag() -> Any:
+    koji_session = koji.ClientSession("https://koji.fedoraproject.org/kojihub")
+    target = koji_session.getBuildTarget("rawhide")
+    return target["build_tag_name"].split("-")[0]
```

Comment:
```suggestion
def get_rawhide_tag() -> str:
    """ Returns the current tag for rawhide, i.e. "f44". """
    koji_session = koji.ClientSession("https://koji.fedoraproject.org/kojihub")
    target = koji_session.getBuildTarget("rawhide")
    return target["build_tag_name"].split("-")[0]
```

I suggest to make the return type annotation `str` because of this:

```python
>>> import koji
>>> koji_session = koji.ClientSession("https://koji.fedoraproject.org/kojihub")
... target = koji_session.getBuildTarget("rawhide")
... type(target["build_tag_name"].split("-")[0])
... 
<class 'str'>
```

Reply:
I had this as returning `str` originally, but the pre-commit job flagged it as an error.  I can try again.